zcash_client_backend/src/data_api.rs

```diff
@@ -1651,17 +1675,17 @@ pub trait WalletWrite: WalletRead {
 
     /// Tells the wallet to track a specific account index for a given seed.
     ///
-    /// Returns details about the imported account, including the unique account  identifier for
-    /// the newly-created wallet database entry, along with the associated  [`UnifiedSpendingKey`].
-    /// Note that the unique account identifier should *not* be  assumed equivalent to the ZIP 32
-    /// account index. It is an opaque identifier for a  pool of funds or set of outputs controlled
-    /// by a single spending authority.  
+    /// Returns details about the imported account, including the unique account identifier for
+    /// the newly-created wallet database entry, along with the associated [`UnifiedSpendingKey`].
+    /// Note that the unique account identifier should *not* be assumed equivalent to the ZIP 32
+    /// account index. It is an opaque identifier for a pool of funds or set of outputs controlled
+    /// by a single spending authority.
     ///
-    /// Import accounts with indexes that are exactly one greater than the highest existing account
-    /// index to ensure account indexes are contiguous, thereby facilitating automated account
+    /// Import accounts with indices that are exactly one greater than the highest existing account
+    /// index to ensure account indices are contiguous, thereby facilitating automated account
     /// recovery.
     ///
-    /// Learn more about account creation and import in the [`WalletWrite`] trait documentation.
+    /// The [`WalletWrite`] trait documentation has more details about account creation and import.
```

Comment:
I think the doc for each of the three account creation/import methods should call out that it fails when there is a duplicate UFVK, even if that's redundant with the `WalletWrite` doc. Note that it is possible to hit this case even if you only use one of the three methods, and there might be implementors who only read the documentation of the methods they are using (because they haven't reached [this page](https://hmpg.net/) yet).